src/py/mat3ra/made/tools/modify.py

```diff
@@ -323,3 +346,55 @@ def condition(coordinate):
     return filter_by_condition_on_coordinates(
         material, condition, use_cartesian_coordinates=use_cartesian_coordinates, invert_selection=invert_selection
     )
+
+
+def add_vacuum(material: Material, vacuum: float = 5.0, add_on_top=True, add_to_bottom=False) -> Material:
+    """
+    Add vacuum to the material along the c-axis.
+    On top, on bottom, or both.
+
+    Args:
+        material (Material): The material object to add vacuum to.
+        vacuum (float): The thickness of the vacuum to add in angstroms.
+        add_on_top (bool): Whether to add vacuum on top.
+        add_to_bottom (bool): Whether to add vacuum on bottom.
+
+    Returns:
+        Material: The material object with vacuum added.
+    """
+    new_material_atoms = to_ase(material)
+    vacuum_amount = vacuum * 2 if add_on_top and add_to_bottom else vacuum
+    ase_add_vacuum(new_material_atoms, vacuum_amount)
+    new_material = Material(from_ase(new_material_atoms))
+    if add_to_bottom and not add_on_top:
+        new_material = translate_to_z_level(new_material, z_level="top")
+    elif add_on_top and add_to_bottom:
+        new_material = translate_to_z_level(new_material, z_level="center")
+    return new_material
+
+
+def remove_vacuum(material: Material, from_top=True, from_bottom=True, fixed_padding=1.0) -> Material:
+    """
+    Remove vacuum from the material along the c-axis.
+    From top, from bottom, or from both.
+
+    Args:
+        material (Material): The material object to set the vacuum thickness.
+        from_top (bool): Whether to remove vacuum from the top.
+        from_bottom (bool): Whether to remove vacuum from the bottom.
+        fixed_padding (float): The fixed padding of vacuum to add to avoid collisions in pbc (in angstroms).
+
+    Returns:
+        Material: The material object with the vacuum thickness set.
+    """
+    atoms = to_ase(translate_to_z_level(material, z_level="bottom"))
```

Comment:
We should not need ASE in this function anymore

Reply:
Still needs to be done

Reply:
Ok